gpflow/utilities/utilities.py

```diff
@@ -91,11 +91,13 @@ def __init__(self):
     return {f".{key.split('.', 1)[-1]}": value for key, value in param_dict.items()}
 
 
-def training_loop(closure: Callable[..., tf.Tensor],
-                  optimizer: Optional[tf.optimizers.Optimizer] = None,
-                  var_list: List[tf.Variable] = None,
-                  maxiter=1e3,
-                  jit=False):
+def training_loop(
+    closure: Callable[..., tf.Tensor],
+    optimizer: Optional[tf.optimizers.Optimizer] = None,
+    var_list: List[tf.Variable] = None,
+    maxiter=1e3,
+    jit=False,
+):
```

Comment:
is this a black formatter change? if so, would you mind getting the changes from develop so that this PR's on topic?